### PR TITLE
Update and add ModMenu things

### PR DESCRIPTION
**Pull request by MODKILLER1001**

### CODE REVIEW

File: src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java [hunk 1]
```diff
@@ -71,6 +72,27 @@ public ConfigScreenFactory<?> getModConfigScreenFactory() {
...
 
+			// Set an option for verifiedIconEnabled
+			visual.addEntry(entryBuilder.startBooleanToggle(Component.translatable("option.NoChatReports.verifiedIconEnabled"), NCRConfig.getClient().verifiedIconEnabled)
+					.setDefaultValue(true)
+					.setTooltip(this.makeTooltip("option.NoChatReports.verifiedIconEnabled.tooltip"))
+					.setSaveConsumer(newValue -> NCRConfig.getClient().verifiedIconEnabled = newValue)
+					.build());
+
+			// Set an option for verifiedIconOffsetX
+			visual.addEntry(entryBuilder.startIntSlider(Component.translatable("option.NoChatReports.verifiedIconOffsetX"), NCRConfig.getClient().verifiedIconOffsetX, -5, 5)
```
Comment (Aizistral) [inline src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java]: Why are offsets limited to 5 pixels?

File: src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java [hunk 2]
```diff
@@ -71,6 +72,27 @@ public ConfigScreenFactory<?> getModConfigScreenFactory() {
...
+					.setDefaultValue(0)
+					.setTooltip(this.makeTooltip("option.NoChatReports.verifiedIconOffsetX.tooltip"))
+					.setSaveConsumer(newValue -> NCRConfig.getClient().verifiedIconOffsetX = newValue)
+					.build());
+
+			// Set an option for verifiedIconOffsetY
+			visual.addEntry(entryBuilder.startIntSlider(Component.translatable("option.NoChatReports.verifiedIconOffsetY"), NCRConfig.getClient().verifiedIconOffsetY, -5, 5)
```
Comment (Aizistral) [inline src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java]: Same as for X option.

**Review by Aizistral — CHANGES_REQUESTED** [review]

File: src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java (hunk 1 above)
Comment (MODKILLER1001) [reply src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java]: Going over 5 had it work half of the time, with the ping check fix I will retest

File: src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java (hunk 2 above)
Comment (MODKILLER1001) [reply src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java]: changed it from -10 to 10

File: src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java [hunk 3]
```diff
@@ -80,14 +80,14 @@ public ConfigScreenFactory<?> getModConfigScreenFactory() {
 					.build());
 
 			// Set an option for verifiedIconOffsetX
-			visual.addEntry(entryBuilder.startIntSlider(Component.translatable("option.NoChatReports.verifiedIconOffsetX"), NCRConfig.getClient().verifiedIconOffsetX, -5, 5)
+			visual.addEntry(entryBuilder.startIntSlider(Component.translatable("option.NoChatReports.verifiedIconOffsetX"), NCRConfig.getClient().verifiedIconOffsetX, -10, 10)
```
Comment (Aizistral) [inline src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java]: What, why so little? Make it 1k at least.

**Review by Aizistral — CHANGES_REQUESTED** [review]

File: src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java (hunk 3 above)
Comment (MODKILLER1001) [reply src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java]: https://user-images.githubusercontent.com/17520451/187282060-47547f7f-145b-4a7a-b237-e07b06080584.mp4

it makes it too hard to control if you have it too much, i could remove the slider and let it be a input

Comment (Aizistral) [reply src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java]: Then make it an input.

Comment (MODKILLER1001) [reply src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java]: Done

**Review by Aizistral — APPROVED** [review]

**Pull request merged**